### PR TITLE
#15857: Binary Forge Sweep Tests Set1

### PR DESCRIPTION
**Pull request by mcw-anasuya**
### Ticket
#15857 

### Problem description
Add forge sweep tests for binary ops

### What's changed
Added automation script to generate forge sweep tests for the following ops:
- add
- minimum
- maximum
- multiply
- subtract
- scatter

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/12415418678)

### CODE REVIEW

**Review by VirdhatchaniKN — APPROVED** [review]

**Review by umadevimcw — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

File: .github/workflows/ttnn-run-sweeps.yaml [hunk 1]
```diff
@@ -244,9 +247,10 @@ on:
           - eltwise.binary.logical_and.logical_and
           - eltwise.binary.logical_and.logical_and_output
           - eltwise.binary.polyval.polyval
-          - eltwise.binary.remainder.remainder
           - eltwise.binary.squared_difference.squared_difference
           - eltwise.binary.squared_difference_output.squared_difference_output
+          - eltwise.binary.scatter.scatter_forge
+          - eltwise.binary.remainder.remainder
```
Comment (jdesousa-TT) [inline .github/workflows/ttnn-run-sweeps.yaml:253]: Where is the associated file to the test?

**Review by jdesousa-TT — CHANGES_REQUESTED** [review]
Please see the comment on `ttnn-run-sweeps.yaml`. Also please test these via pipeline before merge.

File: .github/workflows/ttnn-run-sweeps.yaml (hunk 1 above)
Comment (mcw-anasuya) [reply .github/workflows/ttnn-run-sweeps.yaml:253]: The file is tests/sweep_framework/sweeps/eltwise/binary/remainder/remainder.py
I just grouped the binary remainder tests together for consistency.

Comment (jdesousa-TT) [reply .github/workflows/ttnn-run-sweeps.yaml:253]: Got it, missed the fact that it already existed. Thanks

**Review by jdesousa-TT — APPROVED** [review]

**Pull request merged**